### PR TITLE
Fix FAPI related console UI issues

### PR DESCRIPTION
**Pull request by RivinduM**
### Related Issues
- #17734(https://github.com/wso2/product-is/issues/17734)
- #17515 (https://github.com/wso2/product-is/issues/17515)

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: .changeset/soft-planes-appear.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@wso2is/console": patch
```
Comment (Achintha444) [inline .changeset/soft-planes-appear.md:2]: bump `myaccount` and `i18N`

File: apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx [hunk 2]
```diff
@@ -2156,7 +2173,7 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
                                     value={
                                         initialValues?.accessToken?.validateTokenBinding
                                             ? [ "validateTokenBinding" ]
-                                            : []
+                                            : isFAPIApplication ? [ "validateTokenBinding" ] : []
```
Comment (Achintha444) [inline apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx:2176]: Better to move it to a seperate function and return the relevant from there, to make this more readable.

File: apps/console/src/public/deployment.config.json [hunk 3]
```diff
@@ -924,6 +924,11 @@
                 "disabledFeatures": [],
                 "enabled": true,
                 "scopes": {}
+            },
+            "fapiApplicationCreation": {
```
Comment (brionmario) [inline apps/console/src/public/deployment.config.json:928]: Let's add this to the J2 and add the default value in the framework side as well.

J2: https://github.com/wso2/identity-apps/blob/master/apps/console/java/org.wso2.identity.apps.console.server.feature/resources/deployment.config.json.j2#L1457

Default JSON: https://github.com/wso2/carbon-identity-framework/blob/master/features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json#L1260

File: .changeset/soft-planes-appear.md (hunk 1 above)
Comment (RivinduM) [reply .changeset/soft-planes-appear.md]: added with e2ed10aaa3c1bc125cee4996a20e8ae21906bd9c

File: apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx (hunk 2 above)
Comment (RivinduM) [reply apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx]: fixed with e2ed10aaa3c1bc125cee4996a20e8ae21906bd9c

File: apps/console/src/public/deployment.config.json (hunk 3 above)
Comment (RivinduM) [reply apps/console/src/public/deployment.config.json:928]: Added to J2 with e2ed10aaa3c1bc125cee4996a20e8ae21906bd9c

Comment (RivinduM) [reply apps/console/src/public/deployment.config.json:928]: Added to default json with https://github.com/wso2/carbon-identity-framework/pull/5171

File: modules/i18n/src/translations/en-US/portals/console.ts [hunk 4]
```diff
@@ -1518,16 +1518,16 @@ export const console: ConsoleNS = {
...
                                                 " subject. Subject identifier URI is used in subject value" +
                                                 " calculation."
                                         },
                                         pairwise: {
-                                            label: "pairwise",
-                                            hint: "This option will use the pairwise subject identifier as the" +
-                                                " subject. Subject identifier URI and callback URI or sector" +
-                                                " identifier URI is used in subject value calculation."
+                                            label: "Pairwise",
+                                            hint: "Enable pairwise to assign a unique pseudonymous ID " +
```
Comment (dasuni-30) [inline modules/i18n/src/translations/en-US/portals/console.ts:1528]: Shall we verify this content with the Docs team?

File: apps/console/src/features/applications/components/wizard/minimal-application-create-wizard.tsx [hunk 5]
```diff
@@ -1064,26 +1066,34 @@ export const MinimalAppCreateWizard: FunctionComponent<MinimalApplicationCreateW
...
-                            ] }
-                        />
-                        <Hint compact>
-                            { t("console:develop.features.applications.forms.generalDetails.fields" +
-                                ".isFapiApp.hint" ) }
-                        </Hint>
-                    </div>
+                    {
+                        // The FAPI App creation checkbox is only present in OIDC Standard-Based apps
+                        customApplicationProtocol === SupportedAuthProtocolTypes.OAUTH2_OIDC &&
```
Comment (dasuni-30) [inline apps/console/src/features/applications/components/wizard/minimal-application-create-wizard.tsx:1071]: We can improve formatting here.

customApplicationProtocol === SupportedAuthProtocolTypes.OAUTH2_OIDC
&& selectedTemplate?.name === ApplicationTemplateNames.STANDARD_BASED_APPLICATION
&& isFAPIAppCreationEnabled
&& (

File: apps/console/src/features/applications/components/wizard/minimal-application-create-wizard.tsx [hunk 6]
```diff
@@ -165,7 +166,8 @@ export const MinimalAppCreateWizard: FunctionComponent<MinimalApplicationCreateW
         state.config.ui.isClientSecretHashEnabled);
     const orgType: OrganizationType = useSelector((state: AppState) =>
         state?.organization?.organizationType);
-
+    const isFAPIAppCreationEnabled: boolean = useSelector((state: AppState) =>
+        state.config.ui.features.fapiApplicationCreation.enabled);
```
Comment (dasuni-30) [inline apps/console/src/features/applications/components/wizard/minimal-application-create-wizard.tsx:170]: Shall we add null checks?

```suggestion
        state.config.ui.features?.fapiApplicationCreation?.enabled);
```

File: apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx [hunk 7]
```diff
@@ -1906,160 +1923,168 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
...
+                    </Grid.Column>
+                    <Grid.Column mobile={ 16 } tablet={ 16 } computer={ 16 }>
+                        <Heading as="h4">
+                            { t("console:develop.features.applications.forms.inboundOIDC.sections" +
+                                ".pushedAuthorization.heading") }
+                        </Heading>
+                        <Field
+                            ref={ requirePushedAuthorizationRequests }
+                            name={ "requirePushAuthorizationRequest" }
+                            label=""
```
Comment (dasuni-30) [inline apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx:1940]: Shall we remove this empty prop?

Comment (RivinduM) [reply apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx]: removed with [db451e6](https://github.com/wso2/identity-apps/pull/4616/commits/db451e67d57f674883d760ba9b0831bb14accd8f)

File: apps/console/src/features/applications/components/wizard/minimal-application-create-wizard.tsx (hunk 6 above)
Comment (RivinduM) [reply apps/console/src/features/applications/components/wizard/minimal-application-create-wizard.tsx]: added with [db451e6](https://github.com/wso2/identity-apps/pull/4616/commits/db451e67d57f674883d760ba9b0831bb14accd8f)

File: apps/console/src/features/applications/components/wizard/minimal-application-create-wizard.tsx (hunk 5 above)
Comment (RivinduM) [reply apps/console/src/features/applications/components/wizard/minimal-application-create-wizard.tsx]: fixed with [db451e6](https://github.com/wso2/identity-apps/pull/4616/commits/db451e67d57f674883d760ba9b0831bb14accd8f)

**Review by brionmario — APPROVED** [review]

**Pull request merged**